### PR TITLE
Add Redux repos and standards

### PR DESCRIPTION
**Pull request by Almouro**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/redux-standards/redux/write-your-own-saga-effects.s.md">write-your-own-saga-effects.s.md</a>**: Title doesn't contain `[Standard]` tag at the begining
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/redux-standards/redux/write-your-own-saga-effects.s.md">write-your-own-saga-effects.s.md</a>**: Standard doesn't have a *Checks* part, how could you call this an Standard?
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/redux-standards/redux/write-your-own-saga-effects.s.md">write-your-own-saga-effects.s.md</a>**: You failed to mention a *Bad Examples*
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/redux-standards/redux/write-your-own-saga-effects.s.md">write-your-own-saga-effects.s.md</a>**: You failed to mention a *Good Examples* 
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/redux-standards/redux/write-your-own-saga-effects.s.md">write-your-own-saga-effects.s.md</a>**: Does not seem to be included in the root summary
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: redux/write-your-own-saga-effects.mo.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# [MO] Write your own saga effects *(~3 min)*
+
+## Owner: Alexandre Moureaux
+
+## Prerequisites
+- [ ] Redux installed
```
Comment (tychota) [inline redux/write-your-own-saga-effects.mo.md]: How ? Need some links !

File: redux/write-your-own-saga-effects.mo.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- [ ] Redux saga installed
```
Comment (tychota) [inline redux/write-your-own-saga-effects.mo.md]: Idem

File: redux/write-your-own-saga-effects.mo.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Reproducible example *(~3min)*
```
Comment (tychota) [inline redux/write-your-own-saga-effects.mo.md]: why is it cool ?

why not `yield* authenticatedCall(..)`

**Review by tychota — CHANGES_REQUESTED** [review]

File: redux/write-your-own-saga-effects.mo.md (hunk 1 above)
Comment (Almouro) [reply redux/write-your-own-saga-effects.mo.md]: Really?

File: redux/write-your-own-saga-effects.mo.md (hunk 2 above)
Comment (Almouro) [reply redux/write-your-own-saga-effects.mo.md]: Really?

**Comment by tychota** [comment]
Closing as too old PR. Feel free to reopen later.

This PR use to add a standard but only points to a repository. Please put back a small artcile MO withh a context, and some steps with checks.

**Pull request closed without merging**